### DISCUSSION TITLE
How do I use sass??

### DISCUSSION THREAD

**Discussion by minnieoo** [openstyles/stylus#1708] [General]
i put preprocessor sass, but its throwing errors at sass variables, why??

more importantly, WHERE can i find info on using a preprocessor with stylus?? looked everywhere.

**Comment by tophf** [comment]
It's not supported. You can use `less` or `stylus`, see https://github.com/openstyles/stylus/wiki/Writing-UserCSS#preprocessor

**Reply by minnieoo** [reply]
thank you
